project/AppDependencies.scala

```diff
@@ -20,31 +20,41 @@ import sbt._
 
 object AppDependencies {
 
+  val AkkaVersion = "2.6.20"
+
   val compile = Seq(
     ws,
-    "uk.gov.hmrc"       %% "bootstrap-backend-play-28"         % "6.4.0",
-    "uk.gov.hmrc"       %% "auth-client"                       % "5.7.0-play-28",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"                % "0.70.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-work-item-repo-play-28" % "0.70.0",
-    "uk.gov.hmrc"       %% "play-scheduling-play-28"           % "8.0.0",
-    "uk.gov.hmrc"       %% "domain"                            % "6.2.0-play-28",
-    "net.codingwell"    %% "scala-guice"                       % "5.0.2",
-    "com.typesafe.play" %% "play-json-joda"                    % "2.9.2"
+    "com.typesafe.akka"         %% "akka-actor"                        % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-actor-typed"                  % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-protobuf-v3"                  % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-stream"                       % AkkaVersion,
+    "com.typesafe.akka"         %% "akka-serialization-jackson"        % AkkaVersion,
+    "uk.gov.hmrc"               %% "bootstrap-backend-play-28"         % "6.4.0",
+    "uk.gov.hmrc"               %% "auth-client"                       % "5.7.0-play-28",
+    "uk.gov.hmrc"               %% "time"                              % "3.3.0",
+    "uk.gov.hmrc.mongo"         %% "hmrc-mongo-play-28"                % "0.70.0",
+    "uk.gov.hmrc.mongo"         %% "hmrc-mongo-work-item-repo-play-28" % "0.70.0",
+    "uk.gov.hmrc"               %% "play-scheduling-play-28"           % "8.1.0",
```

Comment:
do we still need this dependency for **play-scheduling-play-28**

Reply:
Great point, totally missed this, thanks Kapil.